src/client/components/SuggestedRoutes/SuggestedRouteDetails.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import TitleAndDescribe from "../TitleAndDescribe";
+
+export default class SuggestedRouteDetails extends Component {
```

Comment:
why this component is class based? why not a functional component? it doesn't  hold any state of it's own or need life cycle hook